README.md

```diff
@@ -1,12 +1,35 @@
-# collaboration
+# Administration for TSC + Community Committee collaborative efforts
+This repository is formal efforts of the Node.js project that are overseen and require input from both the TSC and CommComm. 
 
-***ARCHIVE*: This repository is no longer active. Please see the the [Community Committee](https://github.com/nodejs/community-committee) repository for information on current efforts around supporting and growing the Node.js Community.**
+Initiatives that are overseen here:
+- [Travel Fund](https://github.com/nodejs/admin/blob/master/travel_fund.md)
 
-Facilitating Collaboration among the Node.js Working Groups.
+Repositories that fall under the responsibility of the TSC and CommComm:
 
-This team is just getting started. See [#1 - A Place To Start](https://github.com/nodejs/collaboration/issues/1) to follow the overall planning efforts.
+- this repo
+- (Getting Started)https://github.com/nodejs/getting-started
+- [Collaboration Summit](https://github.com/nodejs/summit)
+- (Moderation](https://github.com/nodejs/moderation)
```

Comment:
Should be: 
```
[Moderation](https://github.com/nodejs/moderation)
```

Reply:
Should note that this is a private repo. We always get reports about 404s otherwise.